src/main/java/org/opentripplanner/middleware/utils/JsonUtils.java

```
@@ -16,15 +15,22 @@
 public class JsonUtils {
 
     private static final Logger LOG = LoggerFactory.getLogger(JsonUtils.class);
-    private static Gson gson = new Gson();
+
+    // Gson is not suitable for parsing OTP itineraries (errors on parsing Dates).
+    // private static Gson gson = new Gson();
+
     private static final ObjectMapper mapper = new ObjectMapper();
 
     /**
      * Serialize an object into a JSON string representation.
-     * TODO: Replace with use of objectmapper?
      */
     public static String toJson(Object object) {
-        return gson.toJson(object);
+        try {
+            return mapper.writeValueAsString(object);
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
+        }
+        return "{}";
```

If an error happens, shouldn't we return an error in JSON?

---

Updated, but not sure on syntax. Will need reviewing.